src/middlewares/natsToRestBridge.ts

```diff
@@ -10,27 +10,34 @@ const jsonCodec = JSONCodec();
 export default function restToNatsBridgeFactory(
     natsConnection: NatsConnection
 ) {
+    const hdr = headers();
```

Comment:
we should probably create a new headers object for each request as opposed to a reused one across requests, so that we don't carry state between requests by accident. let's just move this `const hdr = headers()` inside the handler